data/styles/outdoors/include/priorities_4_overlays.prio.txt

```diff
@@ -966,6 +966,7 @@ amenity-kindergarten                                # icon z17- (also has captio
 amenity-school                                      # icon z17- (also has caption(optional) z17-, area z15-)
 historic-archaeological_site                        # icon z17- (also has caption(optional) z17-)
 historic-boundary_stone                             # icon z16- (also has caption(optional) z16-)
+boundary-marker                                     # icon z16- (also has caption(optional) z16-)
```

Comment:
here it should be moved to priority 220 also

Reply:
Done